hack/test-e2e.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -x
+set -e
+
+ARTIFACT_DIR=${ARTIFACT_DIR:-_output}
+manifest=$(mktemp)
+global_manifest=$(mktemp)
+
+cleanup(){
+  local return_code="$?"
+  set +e
+
+  cat $manifest > $ARTIFACT_DIR/manifest
```

Comment:
* a comment that this is saving manifest in the test results for debugging would be useful.
* nit: `cp` would perhaps look better.
* `rm -f` of the temp files would be great in the end.

Reply:
fixed